### PR TITLE
CloudPods docs revamp

### PR DESCRIPTION
**Pull request by giograno**

### CODE REVIEW

**Comment by thrau** [comment]
@giograno could you rebase to v1 and target the v1 branch?

File: content/en/tools/cloud-pods/pods-cli/index.md [hunk 1]
```diff
@@ -0,0 +1,171 @@
...
+
+For instance, the following command
+```
+localstack config --name my_pod
+```
+will implicitly pass a pod name to all subsequent CLI commands. 
+Such a configuration will be saved locally on the host machine in a JSON file (e.g., in `~/.localstack/cloudpods/pods-config.json`).
+
+## Commands
+
```
Comment (thrau) [inline content/en/tools/cloud-pods/pods-cli/index.md:31]: @whummer  had this wonderful figure showing what components the cli commands interact with. Could we add this here?

File: content/en/tools/cloud-pods/_index.md [hunk 2]
```diff
@@ -0,0 +1,47 @@
+---
+title: "Cloud Pods"
+weight: 10
+categories: ["LocalStack Pro", "Tools", "Persistence"]
+description: >
+  Cloud Pods provides a new way of collaborating in cloud application development workflows.
+---
+
+Cloud Pods are a mechanism that allows you to take a snapshot of your local instance, persist it to a storage backend, and easily share it with your team members.
```
Comment (thrau) [inline content/en/tools/cloud-pods/_index.md:9]:
```suggestion
Cloud Pods are a mechanism that allows you to take a snapshot of the state in your current LocalStack instance, persist it to a storage backend, and easily share it with your team members.
```

**Review by thrau — APPROVED** [review]
looks great! just two minor suggestions

File: content/en/tools/cloud-pods/pods-cli/index.md (hunk 1 above)
Comment (whummer) [reply content/en/tools/cloud-pods/pods-cli/index.md:31]: Done, added it. In the section above, though, seems to fit better in the general description of cloud pods. Post v1 we can still move it down here (at least the right part of the figure that illustrates the pods operations). 👍

**Pull request merged**